### DISCUSSION TITLE
Resources: YouTube channels

### DISCUSSION THREAD

**Discussion by jschuur** [jschuur/decent-nextjs-starter-template#45] [General]
Channels with a lot of Next.js/React content for the Resources section (#5).

Add your sugestions! 

https://www.youtube.com/@ByteGrad
https://www.youtube.com/@DaveGrayTeachesCode
https://www.youtube.com/@WebDevCody
https://www.youtube.com/@joshtriedcoding
https://www.youtube.com/@jherr

**Comment by ghost** [comment]


**Reply by rajeevrj256** [reply]
ans
